commons/ava_networks/test_gecko_network.go

```diff
@@ -178,9 +184,11 @@ func (loader TestGeckoNetworkLoader) ConfigureNetwork(builder *networks.ServiceN
 			loader.bootstrapperSnowSampleSize,
 			loader.bootstrapperSnowQuorumSize,
 			loader.isStaking,
-			bootNodeIds[0:i], // Only the node IDs of the already-started nodes
+			make(map[string]string), // No additional CLI args for the default network
```

Comment:
I think this comment should probably be `No additional CLI args for the bootstrapper nodes` right?

Reply:
Yup, updated this